background.js

```diff
@@ -1,298 +1,323 @@
+/*  Dezoomify-extension – background service-worker (Manifest V3)
+    --------------------------------------------------------------------
+    This is a 1-for-1 port of the original MV2 background script.
+    Main differences:
+      • Runs as a **service-worker** (declared in manifest.json).
+      • All chrome.browserAction.* API calls → chrome.action.* .
+      • Context-menu context changed to "action".
+      • Removed the setInterval heartbeat (worker wakes on events).
+      • getManifest() now returns manifest.action instead of browser_action.
+      • updateStatus() chooses icons from manifest.action.default_icon.
+    Everything else (regexes, request listener, menu actions) is unchanged.
+*/
```

Comment:
This sounds like ai slop. should probably be human-reviewed, and included in the pr comments instead of a comment in the code